base/metrics/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package metrics provides a summary metric implementation
```

Comment:
Please rename to `package summarymetrics`.

Reply:
done